sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -32,10 +32,11 @@
     private static final Tracer TRACER = OpenTelemetry.getTracerFactory().get("Azure-OpenTelemetry");
 
     // standard attributes with AMQP request
-    static final String COMPONENT = "component";
     static final String MESSAGE_BUS_DESTINATION = "message_bus.destination";
     static final String PEER_ENDPOINT = "peer.address";
 
+    static final String AZ_NAMESPACE_KEY = "az.namespace";
```

Comment:
Shouldn't the same constant from the interface this implements be usable instead of having it again?